ImportXml.py

```diff
@@ -0,0 +1,182 @@
+# flake8: noqa
+import re
+import sys
+from json import dump
+
+from bs4 import BeautifulSoup
```

Comment:
I forgot to mention this, but you suppose to add bs4 to dependencies. For that, you need to add packages by `poetry add beautifulsoup4`. So it will be added to `pyproject.toml` and `poetry.lock`

Reply:
ye, thanks, I add it, and update the poetry.